pretix_sepadebit/signals.py

```diff
@@ -59,6 +68,65 @@ def register_csv(sender, **kwargs):
     return DebitList
 
 
+def get_todo_sepa_reminders(date):
+    q_list = []
+
+    for event in Event.objects.all():
```

Comment:
We should avoid looping over all events every few minutes. In some pretix installs, there are tens of thousands of events. See main review comment for an idea on how to approach this.

Reply:
Also, this is missing the check that our plugin is enabled at all, e.g. ``filter(plugins__contains="pretix_sepadebit")``